Doc/library/typing.rst

```diff
@@ -2604,10 +2604,11 @@ Functions and decorators
 
 .. function:: reveal_type(obj, /)
 
-   Reveal the inferred static type of an expression.
+   Ask a static type checker to reveal the statically inferred type of an
+   expression.
 
-   When a static type checker encounters a call to this function,
-   it emits a diagnostic with the type of the argument. For example::
+   When a static type checker encounters a call to this function, it emits a
+   diagnostic with the inferred static type of the argument. For example::
```

Comment:
Tiny nit: I'd prefer to keep a minimal diff here; the first of these two lines here doesn't need to change:

```suggestion
   When a static type checker encounters a call to this function,
   it emits a diagnostic with the inferred static type of the argument.
   For example::
```

Reply:
Done!